src/styles.css

```diff
@@ -1,10 +1,30 @@
+:root {
+  --primary-color: #FBDC57;
+  --secondary-color: black;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --primary-color: black;
+    --secondary-color:#FBDC57;
+  }
+}
+
 body {
-  background: #0e0887;
-  color: white;
+  background-color: var(--primary-color);
 }
 
 a {
-  color: white;
+  color: var(--secondary-color);
+}
+
+a:hover {
```

Comment:
This is the source of a lot of bugs. We need to find a better solution. We need to scope the hover css styles to specific elements.